collector/interrupts_common.go

```diff
@@ -18,19 +18,28 @@
 package collector
 
 import (
+	"github.com/alecthomas/kingpin/v2"
 	"github.com/go-kit/log"
 	"github.com/prometheus/client_golang/prometheus"
 )
 
 type interruptsCollector struct {
-	desc   typedDesc
-	logger log.Logger
+	desc         typedDesc
+	logger       log.Logger
+	nameFilter   deviceFilter
+	includeZeros bool
 }
 
 func init() {
 	registerCollector("interrupts", defaultDisabled, NewInterruptsCollector)
 }
 
+var (
+	interruptsInclude      = kingpin.Flag("collector.interrupts.name-include", "Regexp of interrupts name to include (mutually exclusive to --collector.interrupts.name-exclude).").String()
+	interruptsExclude      = kingpin.Flag("collector.interrupts.name-exclude", "Regexp of interrupts name to exclude (mutually exclusive to --collector.interrupts.name-include).").String()
+	interruptsIncludeZeros = kingpin.Flag("collector.interrupts.include-zeros", "Include interrupts that have a zero value").Default("true").Bool()
```

Comment:
Should this be in the README, too?

Reply:
We don't really have a full flag documentation in the README.